pkg/monitor/agent.go

```diff
@@ -84,20 +82,26 @@ func (monitor *agentMonitor) getAgentCRD() *v1beta1.AntreaAgentInfo {
 func (monitor *agentMonitor) createAgentCRD() (*v1beta1.AntreaAgentInfo, error) {
 	agentCRD := new(v1beta1.AntreaAgentInfo)
 	monitor.querier.GetAgentInfo(agentCRD, false)
+	agentCRD.Name = monitor.querier.GetNodeName()
```

Comment:
Do you think we can set its value in GetAgentInfo() just like we used to do?

Reply:
Right. Set the Node name in GetAgentInfo sounds better to me.

Reply:
OK, will update this part.